Library/Homebrew/env_config.rbi

```diff
@@ -1,7 +1,241 @@
 # typed: strict
 
 module Homebrew::EnvConfig
+  sig { returns(String) }
+  def self.additional_google_analytics_id; end
```

Comment:
Agreed with @Bo98: it'd be really nice to autogenerate these.

Reply:
Yeah, no idea how to do that. Can @Bo98 or @dduugg teach us?

Reply:
Now that 4.0.0 is out I can have a look at this.

Reply:
Will have something to show by Monday. I would point to the existing sorbet/parlour/attr.rb, but `parser` code isn't always the easiest to follow so I might see if I can write that as a runtime generator rather than a static generator.